src/Controller/SaferpayResponseController.php

```diff
@@ -15,7 +15,10 @@
 use Symfony\Component\HttpFoundation\Response;
 
 /**
- * Saferpay response controller.
+ * Class SaferpayResponseController
```

Comment:
the old documentation was actually fine, package and so on looks like phpstorm default documentation.